### PR TITLE
Created IDs PRS13997Valeriani and PRS13998Formigli

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
D. Valeriani wrote in a letter to Formigli informing him about the contents of BMLacq38. This paper letter is attached to BMLacq38. I cannot read the old Italian writing and Marrassini is very short in summarizing the content. This is why I cannot say much about these gentlemen.

### CODE REVIEW

File: new/PRS13997Valeriani.xml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2023-04-18">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person>
+                    <persName xml:lang="gez" xml:id="n1">D. Valeriani</persName>
```
Comment (eu-genia) [inline new/PRS13997Valeriani.xml:48]: lang="it"
split forename (Domenico) surname (Valeriani)
delete normalized

File: new/PRS13997Valeriani.xml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                    <persName xml:lang="gez" type="normalized" corresp="#n1"/>
+                    <nationality type="Italy"/>
+                    <birth/>
+                    <death/>
+                    <floruit>19th century</floruit>
```
Comment (eu-genia) [inline new/PRS13997Valeriani.xml:53]: see preface (Marrassini p. 83)

 Come curiosita si può anche notare la lunga
 lettera, non riportata per esteso nella descrizione del codice, annessa
 sempre all'Acq. e doni 38, in cui un D. Valeriani sfoggia coloritamente
 conoscenze non del tutto esatte sul contenuto del manoscritto; si tratta
 forse del Domenico Valeriani autore, fra l'altro, dell'Atlante del Basso
 e dell'Alto Egitto , stampato a Firenze nel 1837

File: new/PRS13997Valeriani.xml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
```
Comment (eu-genia) [inline new/PRS13997Valeriani.xml:37]: no Gez, It

File: new/PRS13998Formigli.xml [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2023-04-18">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person>
+                    <persName xml:lang="gez" xml:id="n1">Formigli</persName>
```
[the remark this answers is not in the document]
Comment (eu-genia) [reply new/PRS13998Formigli.xml:48]: most probably -
Giuseppe Formigli 
Bookseller, author, and publisher in Canto al Diamante (a district in Florence) in mid-19th century, see e.g. https://books.google.de/books/about/Guida_per_la_citt%C3%A0_di_Firenze_e_suoi_co.html?id=NFlCAAAAYAAJ&redir_esc=y

File: new/PRS13998Formigli.xml [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+                    <persName xml:lang="gez" type="normalized" corresp="#n1"/>
+                    <nationality type="Italy"/>
+                    <birth/>
+                    <death/>
+                    <floruit>19th century</floruit>
```
Comment (eu-genia) [inline new/PRS13998Formigli.xml:53]: notBefore 1800 notAfter 1850 first half of the 19th century

File: new/PRS13997Valeriani.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (eu-genia) [reply new/PRS13997Valeriani.xml:53]: we also need a pointer to `LOC3096Floren `in `floruit`

File: new/PRS13998Formigli.xml (hunk 4 above)
[the remark this answers is not in the document]
Comment (eu-genia) [reply new/PRS13998Formigli.xml:48]: we also need a pointer to `LOC3096Floren `in `floruit`

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/PRS13997Valeriani.xml (hunk 1 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS13997Valeriani.xml:48]: How can I split the name into forename and surname?

File: new/PRS13997Valeriani.xml (hunk 3 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS13997Valeriani.xml:37]: Done.

File: new/PRS13998Formigli.xml [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
```
[the remark this answers is not in the document]
Comment (CarstenHoffmannMarburg) [reply new/PRS13998Formigli.xml:37]: Done.

File: new/PRS13998Formigli.xml (hunk 4 above)
[the remark this answers is not in the document]
Comment (CarstenHoffmannMarburg) [reply new/PRS13998Formigli.xml:48]: Like this: 
`<floruit>19th century in <ptr cert="high" cRef="LOC3096Floren"/></floruit>`

File: new/PRS13997Valeriani.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (CarstenHoffmannMarburg) [reply new/PRS13997Valeriani.xml:53]: Done like this: 
`<floruit notBefore="1800" notAfter="1850">First half of the 19th century in <ptr cert="high" cRef="LOC3096Floren"/>.</floruit>`

File: new/PRS13997Valeriani.xml (hunk 1 above)
Comment (eu-genia) [reply new/PRS13997Valeriani.xml:48]: https://betamasaheft.eu/Guidelines/?q=surname&id=persons
https://www.tei-c.org/release/doc/tei-p5-doc/en/html/examples-forename.html

File: new/PRS13998Formigli.xml (hunk 4 above)
[the remark this answers is not in the document]
Comment (eu-genia) [reply new/PRS13998Formigli.xml:48]: we use `placeName `for places
```
 <occupation type="academic">bookseller, author, and publisher</occupation>
<floruit notBefore="1800" notAfter="1850">mid-19th century in <placeName ref="LOC3096Floren"/></floruit>
```

File: new/PRS13997Valeriani.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (eu-genia) [reply new/PRS13997Valeriani.xml:53]: we use `placeName `for places

File: new/PRS13997Valeriani.xml (hunk 1 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS13997Valeriani.xml:48]: Done.

File: new/PRS13997Valeriani.xml (hunk 2 above)
Comment (eu-genia) [reply new/PRS13997Valeriani.xml:53]: so we need at least addName (Prof), occupation (academic), reference to Florence, dates we know he was active in (1830, 1837) could also be encoded

File: new/PRS13998Formigli.xml (hunk 4 above)
[the remark this answers is not in the document]
Comment (CarstenHoffmannMarburg) [reply new/PRS13998Formigli.xml:48]: I will encode Giuseppe Formigli and will add a cert=high in the manuscript record.

File: new/PRS13997Valeriani.xml (hunk 2 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS13997Valeriani.xml:53]: Like this: 
```
<person>
                    <persName xml:lang="it" xml:id="n1">
                        <addName>Prof</addName><forename>Domenico</forename><surname>Valeriani</surname>
                    </persName>
                    <nationality type="Italy"/>
                    <birth/>
                    <death/>
                    <occupation type="academic"></occupation>
                    <floruit notBefore="1830" notAfter="1837">mid-19th century in <placeName ref="LOC3096Floren"/></floruit>
                </person>
```

File: new/PRS13998Formigli.xml [hunk 7]
```diff
@@ -45,12 +45,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <body>
             <listPerson>
                 <person>
-                    <persName xml:lang="gez" xml:id="n1">Formigli</persName>
-                    <persName xml:lang="gez" type="normalized" corresp="#n1"/>
+                    <persName xml:lang="gez" xml:id="n1"><forename>Giuseppe</forename><surname>Formigli</surname></persName>
```
Comment (eu-genia) [inline new/PRS13998Formigli.xml:48]: as above

File: new/PRS13997Valeriani.xml [hunk 8]
```diff
@@ -45,12 +45,14 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <body>
             <listPerson>
                 <person>
-                    <persName xml:lang="gez" xml:id="n1">D. Valeriani</persName>
-                    <persName xml:lang="gez" type="normalized" corresp="#n1"/>
+                    <persName xml:lang="it" xml:id="n1">
+                        <addName>Prof</addName><forename>Domenico</forename><surname>Valeriani</surname>
```
[the remark this answers is not in the document]
Comment (CarstenHoffmannMarburg) [reply new/PRS13997Valeriani.xml:49]: Done.

File: new/PRS13998Formigli.xml (hunk 7 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS13998Formigli.xml:48]: Done.

File: new/PRS13997Valeriani.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (CarstenHoffmannMarburg) [reply new/PRS13997Valeriani.xml]: Done.

File: new/PRS13998Formigli.xml [hunk 9]
```diff
@@ -45,7 +45,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <body>
             <listPerson>
                 <person>
-                    <persName xml:lang="gez" xml:id="n1"><forename>Giuseppe</forename><surname>Formigli</surname></persName>
+                    <persName xml:lang="gez" xml:id="n1">
```
Comment (eu-genia) [inline new/PRS13998Formigli.xml:48]: it

**Review by eu-genia — CHANGES_REQUESTED** [review]

**Review by eu-genia — APPROVED** [review]

**Review by thea-m — APPROVED** [review]

**Pull request merged**